cmake/templates/BoostConfig.cmake.in

```diff
@@ -150,3 +150,10 @@ endif()
 if(TARGET Boost::chrono)
   hunter_add_rt_library(Boost::chrono)
 endif()
+
+if(TARGET Boost::log AND NOT Boost_USE_STATIC_LIBS)
```

Comment:
You should analyze `BUILD_SHARED_LIBS` instead, it's used in Boost build scheme.

Like this:
```cmake
if(TARGET Boost::log AND "@BUILD_SHARED_LIBS@")
   # ...
endif()
```

Reply:
Is changed.